backup_and_restore/disaster_recovery/about-disaster-recovery.adoc

```diff
@@ -0,0 +1,35 @@
+[id="about-dr"]
+= About disaster recovery
+include::modules/common-attributes.adoc[]
+:context: about-dr
+
+toc::[]
+
+The disaster recovery documentation provides information for administrators on
+how to recover from several disaster situations that might occur with their
+{product-title} cluster. As an administrator, you might need to follow one or
+more of the following procedures in order to return your cluster to a working
+state.
+
+xref:../../backup_and_restore/disaster_recovery/scenario-1-infra-recovery.adoc#dr-infrastructure-recovery[Recovering from lost master hosts]::
+This solution handles situations where you have lost the majority of your master
```

Comment:
@hexfusion Can you review this description for recovering lost master hosts to make sure that it's accurate?

Preview: https://bz-1744569--ocpdocs.netlify.com/openshift-enterprise/latest/backup_and_restore/disaster_recovery/about-disaster-recovery.html

Reply:
This looks good thanks.